package.json

```diff
@@ -221,7 +222,8 @@
     "prettier": "prettier --ignore-path .gitignore \"**/*.{json,md}\"",
     "release:staging": "ts-node ./bin/release_tag.ts staging",
     "release:production": "ts-node ./bin/release_tag.ts production",
-    "start": "yarn build:server && grunt build && concurrently \"yarn start:server:dev\" \"grunt watch\"",
+    "start": "yarn build:server && grunt build && concurrently \"yarn watch:server\" \"grunt watch\"",
+    "watch:server": "tsc-watch -P server/tsconfig.json --onSuccess \"yarn start:server:dev\"",
```

Comment:
Doesn't that start multiple servers on the same port?

Reply:
Ah I see
`Any child process (COMMAND) will be terminated before creating a new one.`